addons/mail/static/src/models/attachment_image_view.js

```diff
@@ -26,7 +33,7 @@ registerModel({
          * @param {MouseEvent} ev
          */
         onClickUnlink(ev) {
-            ev.stopPropagation(); // prevents from opening viewer
```

Comment:
Why not keep the comment?

Reply:
The code tells more now than before since you can follow the handle to the isEvenHandled. Therefore the comment is useless.